cmd/podman/rmi.go

```diff
@@ -58,11 +57,11 @@ func rmiCmd(c *cli.Context) error {
 		return err
 	}
 	removeAll := c.Bool("all")
-	runtime, err := libpodruntime.GetRuntime(c)
+	localRuntime, err := adapter.GetRuntime(c)
```

Comment:
I would just leave this as runtime, err :=

Reply:
You are calling GetRuntime() so call it runtime.  Adding something like activeruntime does not make it any more descriptive to me.  I just think of both as a runtime, and adding more characters to it does not help.  I think you guys are overthinking this.  The runtime is either a libpod runtime or a varlink runtime.

Reply:
ok, done, will catch others in next pr